python/plugins/processing/algs/qgis/ExtentFromLayer.py

```diff
@@ -61,7 +64,7 @@ def svgIconPath(self):
         return QgsApplication.iconPath("/algorithms/mAlgorithmExtractLayerExtent.svg")
 
     def tags(self):
-        return self.tr('polygon,from,vector,raster,extent,envelope,bounds,bounding,boundary,layer').split(',')
+        return self.tr('polygon,from,vector,raster,extent,envelope,bounds,bounding,boundary,layer,round,rounded').split(',')
```

Comment:
Just because you are on it, do you think "from" is an important tag here?

```suggestion
        return self.tr('polygon,vector,raster,extent,envelope,bounds,bounding,boundary,layer,round,rounded').split(',')
```

Reply:
Done